BAZProject/Controllers/TrendingViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  TrendingViewController.swift
+//  BAZProject
+//
+//
+
+import UIKit
+
+class TrendingViewController: UITableViewController {
+    
+    private var moviesListVM: MovieListViewModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setup()
+    }
+
+}
+
+// MARK: - Functions
+
+extension TrendingViewController {
+    
+    /**
+     Configures the view controller by loading trending movies for the day and populating the table view with the results.
+    */
+    private func setup() {
+
+        let trendingMovieDayResource = Resource<MovieList>(url: Endpoint.trendingMovieDay.url) { data in
+            return try? JSONDecoder().decode(MovieList.self, from: data)
+        }
+        
+        MovieAPI().load(resource: trendingMovieDayResource) { (result) in
+            if let movieList = result {
+                self.moviesListVM = MovieListViewModel(movies: movieList.results)
+                self.tableView.reloadData()
+            }
+        }
+    }
+}
+
+// MARK: - TableView's DataSource
+
+extension TrendingViewController {
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return self.moviesListVM?.numberOfRowsInSection(section) ?? 0
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        tableView.dequeueReusableCell(withIdentifier: "TrendingTableViewCell")!
+    }
+
+}
+
+// MARK: - TableView's Delegate
+
+extension TrendingViewController {
+
+    override func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        var config = UIListContentConfiguration.cell()
+        
+        if let movieVM = self.moviesListVM.movieAtIndex(indexPath.row) {
+            config.text = movieVM.title
+            
+            if let url = URL(string: movieVM.poster_path), let data = try? Data(contentsOf: url) {
+                // TODO: pending finish implementing the ImageLoader class with async
```

Comment:
Consejo: Podrías utilizar un warning para marcar al momento de compilar, todos los todo's que te queden pendientes y así no dejar algun remanente.

```suggestion
                #warning(TODO: pending finish implementing the ImageLoader class with async)
```